libraries/unit_converter.rb

```diff
@@ -0,0 +1,40 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: unit_converter.rb
+#
+# Module contains different functions used to manipulate the units of file sizes.
+#
+module CernerSplunk
+  SIZE_SCALE = %w(KB MB GB TB).freeze
+  # Methods converts file sizes in KB, MB, GB and TB into Bytes.
+  def self.convert_to_bytes(string)
+    fail "Unparsable size input: #{string}" if string.match(/^\d+\w{2,3}$/).nil?
+
+    size = string.scan(/\d+|\w/)[0]
+    size_unit = string.scan(/\d+|\w/)[1].upcase
```

Comment:
Had few of the requirements for the inputs specified in the databag docs (like the input needs to have a unit), but I did miss few things that I should have handled. I have fixed the issues you identified. Also update the databag docs.
8ef71dc07ad3750b942bd9a7ce359a74298db128